src/components/LeftSidebar/NewConversation/SearchBox.vue

```diff
@@ -27,8 +27,8 @@
 			v-model="localValue"
 			class="app-navigation-search__input"
 			type="text"
-			:autofocus="autoFocused"
-			:placeHolder="placeholderText">
+			:placeHolder="placeholderText"
+			v-focus>
```

Comment:
Actually this should not be focused, but the NewMessageForm (at least when it is displayed)

Reply:
Yep, that's the reason I started this pr